prisma/schema.prisma

```diff
@@ -105,3 +106,8 @@ model Codechallenge {
   createdAt                 DateTime      @default(now())
   expireAt                  DateTime
 }
+
+model Batch{
+  id          Int           @id @default(autoincrement())
+  batch       String        @unique
+}
```

Comment:
oh you're using this to store all the batch and year, then its better to create new table to store batch ? (many to many relations)

Reply:
Not needed imo, this is just to store the available batches from which the user can choose from.

Reply:
storing same string/data(s) for each user is not best practice, using many-many relation will be apt here.
but anything is fine